src/nat/middleware/red_teaming/register.py

```
@@ -0,0 +1,41 @@
+# limitations under the License.
+"""Registration module for red teaming middleware."""
+
+from __future__ import annotations
+
+from nat.cli.register_workflow import register_middleware
+from nat.middleware.red_teaming.red_teaming_middleware import RedTeamingMiddleware
+from nat.middleware.red_teaming.red_teaming_middleware_config import RedTeamingMiddlewareConfig
+
+
+@register_middleware(config_type=RedTeamingMiddlewareConfig)
+async def red_teaming_middleware(config: RedTeamingMiddlewareConfig, builder):
```

Ruff (0.14.8)

Unused function argument: `builder`

(ARG001)